src/CLICommand/cli_command.ts

```diff
@@ -46,14 +44,10 @@ function setCommanderCommand(commandDescriptor: CommandDescriptor, program: CliA
 			command.option(...optionArguments);
 		}
 	});
-	command = command.action(async (options) => {
-		await (async function () {
-			assertConjunctionParameters(commandDescriptor, options);
-			const exitCode = await commandDescriptor.action(options);
+	command = command.action((options) => {
+		assertConjunctionParameters(commandDescriptor, options);
+		commandDescriptor.action.trigger(options).then((exitCode) => {
 			exitProgram(exitCode || 0);
```

Comment:
Nit: ?? vs ||

Reply:
`??` checks for nullability. `exitCode` actually is of type `number | void` and `void != null` so the linter gets mad

Reply:
```
Argument of type 'ActionReturnType' is not assignable to parameter of type 'number'.
  Type 'void' is not assignable to type 'number'.
```